pf9/cluster/commands.py

```diff
@@ -285,6 +300,7 @@ def create(ctx, **kwargs):
         logger.exception("Cluster Create Failed")
         click.secho("Failed to create cluster {}. {}".format(
                     ctx.params['cluster_name'], e.msg), fg="red")
+        SegmentSessionWrapper(ctx).send_track_error('Cluster Create', e)
```

Comment:
Shouldn't this be sufficient to track all errors? Why do we need an error track to be added into the attach_cluster method, for example.